<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>availableRoomsWindow</class>
 <widget class="QMainWindow" name="availableRoomsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
          <item>
           <widget class="QDateEdit" name="from_dateEdit">
            <property name="displayFormat">
             <string>yyyy-MM-dd</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="till_dateEdit">
            <property name="displayFormat">
             <string>yyyy-MM-dd</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="search_pushButton">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="RightPanel_verticalLayout">
        <item>
         <widget class="QGroupBox" name="verticalGroupBox">
          <property name="title">
           <string>Room information</string>
          </property>
          <layout class="QVBoxLayout" name="roomInformation_verticalLayout">
           <item>
            <layout class="QFormLayout" name="roomInfo_formLayout">
             <item row="2" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Room name:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="roomName_lineEdit">
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>Room name</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Number of beds:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="numberOfBeds_lineEdit">
               <property name="placeholderText">
                <string>Beds number</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Room type:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="roomType_lineEdit">
               <property name="placeholderText">
                <string>Room type</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Price:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Room number:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="roomNumber_lineEdit">
               <property name="placeholderText">
                <string>Room number</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="price_lineEdit">
               <property name="placeholderText">
                <string>Price</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="balcony_checkBox">
             <property name="text">
              <string>Balcony</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Additional notes:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="additionalNotes_plainTextEdit"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="bookButton_horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="book_pushButton">
            <property name="text">
             <string>Book</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>773</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionManage_reservations"/>
    <addaction name="actionEdit_users__data"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionManage_reservations">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Manage reservations</string>
   </property>
  </action>
  <action name="actionEdit_users__data">
   <property name="text">
    <string>Edit users' data</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
